node_modules/oae-preview-processor/lib/api.js

```diff
@@ -507,18 +507,18 @@ var _generatePreviews = function(ctx, callback) {
         if (err) {
             return callback(err);
         } else if (processor) {
-            processor.generatePreviews(ctx, ctx.content, function(err, ignored) {
+            processor.generatePreviews(ctx, ctx.content, function(err, status) {
                 if (err) {
-                    ctx.setStatus('error', callback);
-                } else if (ignored) {
-                    ctx.setStatus('ignored', callback);
+                    ctx.setStatus(ContentConstants.previews.ERROR, callback);
```

Comment:
This is the error condition - if there was an error in the preview processing, we should set status to 'error', correct?